sources/packages/api/src/route-controllers/education-program-offering/education-program-offering.controller.ts

```diff
@@ -253,12 +260,15 @@ export class EducationProgramOfferingController {
     @Param("locationId") locationId: number,
     @Param("programId") programId: number,
     @Param("programYearId") programYearId: number,
+    @Query("includeInActivePY") includeInActivePY?: boolean,
   ): Promise<OptionItem[]> {
     const offerings =
       await this.programOfferingService.getProgramOfferingsForLocation(
         locationId,
         programId,
         programYearId,
+        undefined,
+        includeInActivePY,
```

Comment:
Can we please remove the use of undefined, so that change the signature of getProgramOfferingsForLocation, to have mandatory includeInActivePY.